test/CadlRanchProjects.Tests/type-property-optionality.cs

```diff
@@ -14,6 +15,8 @@ namespace CadlRanchProjects.Tests
 {
     public class TypePropertyOptionalTests : CadlRanchTestBase
     {
+        private static readonly DateTimeOffset PlainDateData = new DateTimeOffset(2022, 12, 12, 0, 0, 0, 0, new TimeSpan());
```

Comment:
Same

Reply:
removed

Reply:
What did you remove?

Reply:
This part of the code is used in Cadl Ranch and should be removed in nonazure. I mixed up the replies for these two parts. The related changes have been correctly applied.